src/main/perl/TextRender.pm

```diff
@@ -154,16 +153,10 @@ Java properties format (using C<Config::Properties>),
 
 .INI format (using C<Config::Tiny>)
 
-=item general
-
-(using C<Config::General>).
-
-Caution: there is no way to garantee reproducible results as
-there is no control over the order of hash keys. It is recommended to
-try to generate this format in another way (e.g. via TT).
-
 =back
 
+(Builtin module <general> is provided as part of L<CCM::TextRender> since 15.10).
```

Comment:
well, `general` is now an alias for `CCM/general.tt` (https://github.com/quattor/CCM/pull/73/files#diff-3a3731fb118515c51d43b8562ff39e47R316), so if you're using CCM::TextRender (like metaconfig and the test framework), nothing should have changed.

the pod of CCM::TextRender now explains what general is. Maybe i should drop the word `buitin`?